rb_tree.py

```diff
@@ -35,29 +37,34 @@ def __eq__(self, other):
         if self.parent is None or other.parent is None:
             parents_are_same = self.parent is None and other.parent is None
         else:
-            parents_are_same = self.parent.value == other.parent.value and self.parent.color == other.parent.color
```

Comment:
Let's not alter the formatting of unrelated code for this PR specifically

Reply:
I just changed this specific line to fix the pep8 problem. But now, It was changed and I believe that is working perfectly.